templates/confirmation/notify_change_in_email_body.txt

```diff
@@ -0,0 +1,14 @@
+Hi there,
+
+The email address associated with your Zulip account was changed recently. If
+you did this then ignore this email otherwise contact us immediately.
```

Comment:
subject: [Zulip] Email address changed for {{ realm.name }}

Hi,

We just wanted to let you know that the email associated with your Zulip account was recently changed to {{ new_email }}. If you did not request this change, please contact us immediately at <{{ support_email }}>.

Best,

The Zulip Team